### PR TITLE
DDF-3021 Removed the kernel module, and relocated the branding modules

### PR DESCRIPTION
**Pull request by emmberk**
#### What does this PR do?
This PR rebases and fixes the build for https://github.com/codice/ddf/pull/1994.
#### Who is reviewing it? 
@lessarderic @coyotesqrl @clockard @emmberk @vinamartin
#### Select relevant component teams: 
@codice/build 
#### Choose 2 committers to review/merge the PR.
@pklinef
@stustison
#### How should this be tested?
- Full build
- Itests
- Manual install
#### Any background context you want to provide?
The kernel module was intended to be used as a building block for DDF-based distributions and is no longer necessary.
#### What are the relevant tickets?
[DDF-3021](https://codice.atlassian.net/browse/DDF-3021)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by emmberk** [comment]
build now

**Comment by emmberk** [comment]
build now

File: distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg [hunk 1]
```diff
@@ -21,7 +21,6 @@
 # Comma separated list of features repositories to register by default
 #
 featuresRepositories= \
- mvn:org.codice.ddf/kernel/${project.version}/xml/features, \
```
Comment (vinamartin) [inline distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg]: ❓  Would you also need to get rid of kernel as a boot feature here or no?

Comment (emmberk) [reply distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg]: I still want to start the `kernel` feature which is now defined in the platform-app features repository. This PR deletes the kernel features repository.

Comment (vinamartin) [reply distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg]: Ahh okay, confusing.

**Comment by tbatie** [comment]
We've started using the kernel distribution in our integration tests, if this distribution were to go away we would be forced to have a dependency on all of DDF. I'm hoping we can have more conversation about testing strategies and how DDF should be assembled before this PR is merged.

**Pull request closed without merging**